tensorflow/tools/ci_build/update_version.sh

```diff
@@ -92,6 +93,16 @@ check_existence file "${README_MD}"
 
 sed -i -r -e "s/${OLD_MAJOR}\.${OLD_MINOR}\.([[:alnum:]]+)-/${MAJOR}.${MINOR}.${PIP_PATCH}-/g" "${README_MD}"
 
+# Update the install md files
+NEW_PIP_TAG=$MAJOR.$MINOR.$PIP_PATCH
+OLD_PIP_TAG=$OLD_MAJOR.$OLD_MINOR.$OLD_PIP_PATCH
+
+for file in ${TF_SRC_DIR}/docs_src/install/install_{linux,mac,windows,sources}.md
```

Comment:
For this release, they may be maintained manually.
But eventually they will be a part of the release
@asimshankar for verification.

As they already have 1.1.0 in their versions, until our "old" version becomes 1.1.0, they wont be updated by the script's sed commands. So I think it is safe to incorporate them here now.